esx_service/cli/vmdkops_admin_test.py

```diff
@@ -721,11 +733,13 @@ def test_tenant_access(self):
         # change allow_create to True
         # change max_volume size to 1000MB
         # change total_volume size to 2GB
+        volume_maxsize_in_MB = convert.convert_to_MB("1000MB")
+        volume_totalsize_in_MB = convert.convert_to_MB("3GB")
```

Comment:
Same comment on making test values global or even place all these test values in a separate file thats can be changed by user to configure the test as needed.